lib/premailer/rails/css_helper.rb

```diff
@@ -3,16 +3,21 @@ module Rails
     module CSSHelper
       extend self
 
-      @cache = {}
-      attr :cache
-
       STRATEGIES = [
-        CSSLoaders::CacheLoader,
         CSSLoaders::FileSystemLoader,
         CSSLoaders::AssetPipelineLoader,
-        CSSLoaders::NetworkLoader
+        CSSLoaders::NetworkLoader,
       ]
 
+      def reset_cache!
+        @cache = {}
+      end
+
+      def cache
+        reset_cache! if @cache.nil?
+        @cache
+      end
+
```

Comment:
Cache was spread through two different places, the old `CacheLoader` and this module, which kinda breaks encapsulation. Now cache is fully implemented here and a little bit more robust.